docs/GDI-SOP_review-checklist.md

```diff
@@ -0,0 +1,80 @@
+# SOP Review Checklist
+
+This checklist serves as a guide for **SOP reviewers, approvers, and authorizers**. It outlines the steps and key points to ensure that the SOPs are complete, clear, and compliant with the GDI framework. All reviewers should ensure each point is met before moving on to the next step. Approvers and authorizers have additional responsibilities as noted below.
+
+## General SOP Review Checklist
+
+### 1. **Document Structure & Formatting**
+- [ ] **Compliance with SOP template**: Ensure that the SOP complies with the latest SOP template (see [``GDI-SOP_sop-template.md``](./GDI-SOP_sop-template.md)). Annotations in the template (between ``<>``) contain useful information and formatting rules, including:
+    - **Required Sections**: Ensure the SOP includes the required sections (e.g., ``Index`` or ``Procedures``).
+    - **Linting Process**: Check that key values (e.g., the SOP title, the Template SOP number...) make sense and comply with the expected format.
+- [ ] **Document Consistency**: Check for consistent formatting, structure, and numbering throughout the SOP.
+- [ ] **Metadata**: Verify that metadata in all tables (e.g., versioning, dates, authors...) is included and accurate.
+
+#### Linting Script
+To ease the document linting process, the script ``sop_linter.py`` was created. This script **checks multiple parts of the format of an SOP**, and retrieves any given errors.
+
+You can **manually execute** the script to ensure a file (i.e., the SOP document) passes the linting checks as follows:
+````
+cd standard-operating-procedures
+pip install --upgrade --no-deps -r requirements.txt
+python3 scripts/sop_linter.py tests/GDI-SOP0000_sop-template_for_linting.md -v 1
+````
+
+Replace the test SOP (``tests/GDI-SOP0000_sop-template_for_linting.md``) with the file path of the SOP you are reviewing.
+
+If there are no linting errors, the output will look like the following:
+````
+{
+  "tests/GDI-SOP0000_sop-template_for_linting.md": {
+    "errors": [],
+    "warnings": []
+  }
+}
+````
+
+For your convenience, the script is automatically executed (via [``lint_sops.yml``](../.github/workflows/lint_sops.yml) workflow) whenever a [Pull Request](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/pulls) (PR) containing SOPs is created. Therefore, if you are reviewing a PR, you can **easily see if the SOPs comply with the expected format** by ensuring that the **latest commit has the green check mark** (✔️) next to it.
+
+![Valid: workflow check in Pull Request](images/GDI-SOP_review-checklist-workflows.png)
+
+If, on the contrary, the latest commit has a red cross (❌), some of the checks did not pass and it requires further inspection.
+
+### 2. **Content Completeness**
```

Comment:
Do you mean whole sections that are required? I added that but in the 1st section:
```
    - **Required Sections**: Ensure the SOP includes the required sections (e.g., ``Index`` or ``Procedures``).
```
Also had these rules automated in the section ``Linting Script``. Would you add also the list of "This section, this section, and this section, are required"?

Reply:
Ideally we would have all these things automated, so that the rules of "what is required" live inside the scripts I made to automate it, and that way no human has to go and check every bit that is required.